builtin/game/misc.lua

```diff
@@ -125,6 +125,19 @@ function core.get_player_radius_area(player_name, radius)
 end
 
 
+function core.is_valid_pos(pos)
+	local mapgen_limit = tonumber(core.settings:get("mapgen_limit"))
+	if pos then
+		for _, v in pairs({"x", "y", "z"}) do
+			if not pos[v] or pos[v] < -mapgen_limit or pos[v] > mapgen_limit then
+				return
```

Comment:
Is it better to explicitly return `false`? I don't see any reason to differentiate `false` from `nil` for this method, but it's probably better to return `false` for the sake of consistency and future-proofing (maybe a future addition to this method would want to return `nil`)

Reply:
I agree 'false' is better.

Reply:
`false` isn't equal to `nil` (it's not even the same type similarly, `0` isn't equal to false).
So if someone assumes the function returns `false` on failure as they might considering it returns true on success they will be confused.